tests/NRedisStack.Tests/Search/SearchTests.cs

```diff
@@ -1363,7 +1377,13 @@ public void TestExplainCli()
             .AddTextField("f3", 1.0);
         ft.Create(index, FTCreateParams.CreateParams(), sc);
 
-        var res = ft.ExplainCli(index, new Query("@f3:f3_val @f2:f2_val @f1:f1_val"));
+        var query = "@f3:f3_val @f2:f2_val @f1:f1_val";
```

Comment:
I think yo umisunderstood. This query is used on linues:

1357
1380
1403
1426

let's have them defined once.